spec/author_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rspec'
+require_relative '../author/author'
```

Comment:
Please check the unit tests as they are failing when I run `rspec`

![image](https://user-images.githubusercontent.com/41517876/219126927-476049f3-0382-41bf-ba56-54ff8a1ba63d.png)